README.md

```diff
@@ -25,10 +25,23 @@ It’s a Rails app that:
 
    ```sh
    $ bundle install
+   ```
+
+5. Create development database 
+
+   ```sh
+   $ createdb webpage-versions-db_development
+   ```
+
+6. Edit the file ~\web-monitoring-db\config\database.yml. Uncomment line 35 and append your postgres database role password	
```

Comment:
Hmmm, we really shouldn’t suggest people modify a file managed by version control. We should add support for some other method of configuration (e.g. environment vars, a local config, etc) and recommend that.

The easiest solution (but least nice) is to just tell people to set the `DATABASE_URL` env var when running the rails server:

```sh
$ DATABASE_URL='postgresql://user:password@localhost/my_database' rails server
```

Alternatively, some good gems to help with this are [Figaro](https://github.com/laserlemon/figaro) and [dotenv](https://github.com/bkeepers/dotenv)